gips/0080-optimistic-ebo.md

```diff
@@ -0,0 +1,237 @@
+- **Liveness risks:** If the owner fails to update the EBO, indexers won’t have a reference block to build their POIs, preventing them from closing their allocations or risk slashing if they register a POI for an incorrect block. The system must be redundant to mitigate this risk.
+- **Fork risks:** There might be uncertainty around the correct fork choice when updating the EBO since it is not a fork oracle. For future verifiability efforts, it will likely be necessary to include blockhash in addition to the block number in the EBO’s output, making this even more important.
+
+# **High-Level Description**
+
+## Optimistic EBO Solution Considerations
+
+The goal of this solution is to decentralize the EBO while keeping the structure open for future modifications.
+
+Key improvements:
+
+- **Optimistic operators:** Implementing an optimistic oracle will allow EBO operations to be open and permissionless, improving decentralization, redundancy, and scalability.
+- **Timestamp comparison:** Instead of querying each RPC when detecting a new epoch, a delay is introduced to compare timestamps, reducing time sensitivity and improving the precision of fork selection.
+- **Escalating bond mechanism**: The modularity of the optimistic oracle allows for an escalating bond dispute system, reducing Arbitrator intervention and enabling lower initial stakes without compromising security.
+    - The Arbitrator multisig will act as the ultimate source of truth in the case of disputes in this initial phase, with potential future improvements.
+
+### Optimistic Design
+
+![image.png](../assets//gip-0080/Optimistic%20EBO.png)
+
+### **Optimistic operators**
+
+The optimistic approach allows off-chain agents to propose updates to the Optimistic EBO, which other participants can dispute by staking tokens or using alternative resolution mechanisms to align incentives and ensure accuracy.
+
+For this, we will use [Prophet](https://github.com/defi-wonderland/prophet-modules/tree/dev), an optimistic, modular, and public-good oracle that abstracts the complexity of the proposal and dispute process while offering the flexibility to customize its functionality as needed. For more details on Prophet, refer to the documentation [here](https://docs.prophet.tech/content/modules/index.html).
+~~~~
+The process would work as follows: 
+
+- Any staked actor can propose an update for the EBO, which is published in a contract and remains open for a set period to allow for potential disputes.
+- Any staked actor can also challenge the proposed update. If a dispute arises, the initial proposal is discarded, and a new one can be submitted.
+- If a dispute occurs, the resolution mechanism is triggered, and the winner of the dispute will receive the counterparty’s bond as a reward.
```

Comment:
To avoid certain attack vectors, this should be not the full bond, but half of it?

Reply:
Which attack vectors? What do we do with the other half?